share/gpodder/extensions/enqueue_in_mediaplayer.py

```diff
@@ -81,6 +81,122 @@ def open_files(self, filenames):
             subprocess.Popen(cmd)
 
 
+class MPRISResumer(FreeDesktopPlayer):
+    """
+    resume episod playback at saved time
+    """
+    OBJECT_PLAYER = '/org/mpris/MediaPlayer2'
+    OBJECT_DBUS = '/org/freedesktop/DBus'
+    INTERFACE_PLAYER = 'org.mpris.MediaPlayer2.Player'
+    INTERFACE_PROPS = 'org.freedesktop.DBus.Properties'
+    SIGNAL_PROP_CHANGE = 'PropertiesChanged'
+    NAME_DBUS = 'org.freedesktop.DBus'
+
+    def __init__(self, slug, application, command, bus_name):
+        super(MPRISResumer, self).__init__(slug, application, command)
+        self.title = '/'.join((_('Resume in'), application))
+        self.bus_name = bus_name
+        self.player = None
+        self.position = None
```

Comment:
To make it more obvious that the position is measured in microseconds (according to your comment below), rename this variable to `position_us`.